examples/common/mqtt_agent/mqtt_agent_task.c

```diff
@@ -180,6 +180,23 @@
  */
 #define mqttexampleEVENT_BITS_ALL    ( ( EventBits_t ) ( ( 1ULL << MQTT_AGENT_NUM_STATES ) - 1U ) )
 
+
```

Comment:
Please consolidate the ALPN choices and the decision for which ALPN to use in a single place.

Reply:
Adjusted as requested. Please let me know what you think.